system/View/View.php

```diff
@@ -188,6 +188,14 @@ public function render(string $view, array $options = null, $saveData = null): s
 		$output = ob_get_contents();
 		@ob_end_clean();
 
+		// TODO: Check response Content-Type? Is here the correct place to do it?
+		// Compare Environment or some DebugBar Config?
+		if (ENVIRONMENT == 'development')
+		{
+			$output = '<div class="debug-view"><div class="debug-view-path">' . $file . '</div>'
```

Comment:
This should have an inline style set to `display: none` so that in case someone is in debug mode and has the Toolbar filter turned off, these don't get displayed.